contacts/migrations/0003_alter_contact_phone.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1.1 on 2022-10-14 18:08
```

Comment:
Try deleting all migration files and leaving your DB, and restart migrations again, this way you will have only one migration and not more